<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgWelcome</class>
 <widget class="QDialog" name="DlgWelcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车轮磨损检测系统</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="btnMoniteRealTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>实时监控</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XXX地铁总公司</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnHistDatas">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>历史数据</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDevCom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>设备通讯</string>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnParaSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
  <zorder>btnMoniteRealTime</zorder>
  <zorder>btnExit</zorder>
  <zorder>btnHistDatas</zorder>
  <zorder>btnDevCom</zorder>
  <zorder>btnParaSetting</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
